packages/data-manipulation/html/src/html.ts

```
@@ -0,0 +1,60 @@
+ * Function overload for HTML with escaping control.
+ * @param value The HTML string to process
+ * @param escape If false, returns HTML as-is (XSS-safe). If true, escapes HTML.
+ * @returns The processed HTML string
+ * @example
+ * html('<div></div>', false)
+ * // => '<div></div>'
+ * @example
+ * html('<script>alert("xss")</script>', true)
+ * // => '&lt;script>alert("xss")&lt;/script>'
+ */
+function html(value: string, escape?: boolean): string;
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)